### PR TITLE
Add signed uploads

### PR DESCRIPTION
**Pull request by bautrukevich**

### CODE REVIEW

**Comment by bautrukevich** [comment]
@dmitry-mukhin check PR, please

File: src/Uploadcare/Signature/SecureSignature.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    /**
+     * @var DateTime
+     */
+    private $expire;
+
+    /**
+     * SecureSignature constructor.
+     *
+     * @param string $secretKey
+     * @param int $expireTimeInSeconds
```
Comment (dmitry-mukhin) [inline src/Uploadcare/Signature/SecureSignature.php]: expiretimeinseconds seems to elaborate var name.
if you need more context, doc string is there to help.
```suggestion
     * @param int $lifetime
```

**Comment by bautrukevich** [comment]
> please don't mix formatting and new functionality changes.
ok, I will try, sorry

File: src/Uploadcare/Signature/SecureSignature.php (hunk 1 above)
Comment (bautrukevich) [reply src/Uploadcare/Signature/SecureSignature.php]: Thank you! Fixed

**Comment by bautrukevich** [comment]
@dmitry-mukhin bump

File: src/Uploadcare/Signature/SecureSignature.php (hunk 1 above)
Comment (dmitry-mukhin) [reply src/Uploadcare/Signature/SecureSignature.php]: change the name everywhere, it doesn't make sence

**Comment by bautrukevich** [comment]
Prior to #120

**Pull request closed without merging**